src/radical/pilot/agent/launch_method/srun.py

```diff
@@ -93,7 +93,7 @@ def can_launch(self, task):
     def get_slurm_ver(self):
 
         version = self._version.strip('\n')
-        major_version = eval(version.strip('\n slurm').strip(".")[:2])
+        major_version = int(version.strip('\n slurm').strip(".")[:2])
```

Comment:
This will break in 43 years when slurm reaches version 100.0.0 :-P  Maybe
```
major_version = self._version.split('.')[0].split()[-1]
```
(take element before first dot, pick last word (strip `slurm ` and whatever they decide to print up front)?

Reply:
Even better! thanks for the suggestion. I have committed the change accordingly.